fuse/dir.go

```diff
@@ -1,3 +1,5 @@
+// +build !openbsd
```

Comment:
Adding `// +build !windows` (as a separate line) will do the same for Windows. So while you are at it we might add it as well.